core/storage/storage_models_test.py

```diff
@@ -19,10 +19,19 @@
 
 import inspect
 
+from core.domain import takeout_service
 from core.platform import models
 from core.tests import test_utils
 
-(base_models,) = models.Registry.import_models([models.NAMES.base_model])
+(
+    base_models, collection_models, email_models,
+    exploration_models, feedback_models, skill_models, topic_models,
+    suggestion_models, user_models, story_models, question_models,
```

Comment:
Please have line breaks match those of the list below.

Reply:
Done.